packages/jest-jasmine2/src/setup-jest-globals.js

```diff
@@ -34,21 +34,54 @@ const addSuppressedErrors = result => {
   }
 };
 
+const addPassedExpectations = result => {
+  const {passedExpectations} = getState();
+  setState({passedExpectations: []});
+  result.passedExpectations = result.passedExpectations.concat(passedExpectations);
```

Comment:
do we need to report passed expectations to jasmine? in jest we throw on all assertions and it doesn't really matter